odd-platform-api/src/test/resources/scripts/createAll.sql

```diff
@@ -0,0 +1,393 @@
+create table flyway_schema_history
```

Comment:
why do we need it? Why can't we just use existing flyway migrations for creating db schema?